asset/quickstart/batch-get-effective-iam-policies/main.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_batch_get_effective_iam_policies]
```

Comment:
Looks like this region tag doesn't exist yet in devrel/ , you should create it.

Reply:
I granted our team group the DOC_OWNER and SAMPLE_OWNER permission but still not able to add the region tag to Cloud Asset Inventory product in https://devrel.corp.google.com/snippets/region-tags/new. How long should I wait for permission propagation or did I grant the wrong permission?

Reply:
From the snippet-bot comment above, it seems to indicate the tag will be auto created, let me know if I need to maunually do it and how. Thanks

Reply:
It looks like after permission is propageted, I was able to create the tag: https://devrel.corp.google.com/snippets/region-tags?region_tag=%25asset_quickstart_batch_get_effective_iam_policies%25.